test/replays/132/replays.test.ts

```diff
@@ -184,18 +193,48 @@ it("should return chat mode types correctly", async () => {
 });
 
 it("should handle a netease replay with rogue playerId 3 CommandDataBlocks correctly", async () => {
+  spiedConsoleError = spyConsoleError();
+
   const test = await Parser.parse(path.resolve(__dirname, "moju_vs_fly.nwg"));
+
   expect(test.players).toMatchSnapshot();
+  expect(spiedConsoleError.mock.calls).toHaveLength(2);
```

Comment:
Can we use the jest expectations here?

So that means replacing this with 

```
expect(spy).toHaveBeenCalledTimes(2)
expect(spy).toHaveBeenCalledWith("...") // or NthCalledWith
expect(spy).toHaveBeenCalledWith("...")
```